apps/drec-api/src/pods/issuer/issuer.service.ts

```diff
@@ -112,6 +119,37 @@ export class IssuerService {
     return await this.issueCertificate(issuance);
   }
 
+  private async handleLeftoverReads(
+    group: DeviceGroup,
+    totalReadValueW: number,
+  ): Promise<number> {
+    // Logic
+    // 1. Get the accummulated read values from devices
+    // 2. Transform current value from watts to kw
+    // 3. Add any leftover value from group to the current total value
+    // 4. Separate all decimal values from the curent kw value and store it as leftover value to the device group
+    // 5. Return all the integer value from the current kw value (if any) and continue issuing the certificate
+
+    const totalReadValueKw = group.leftoverReads
+      ? totalReadValueW * 10 ** -3 + Number(group.leftoverReads)
```

Comment:
Why just not divide by `10 ** 3` ?

Reply:
Not sure. But It would look better if I divide it